manifests/mod/authz_default.pp

```diff
@@ -1,3 +1,9 @@
-class apache::mod::authz_default {
-  ::apache::mod { 'authz_default': }
+class apache::mod::authz_default(
+    $apache_version = $::apache::apache_version
+) {
+  if versioncmp($apache_version, '2.4') >= 0 {
```

Comment:
Could you remove the parameter and just reference $::apache::apache_version directly? I don't think there is any reason for anyone to pass the version in, is there?